Client/Coordinators/Launch/LaunchCoordinator.swift

```diff
@@ -11,7 +11,7 @@ protocol LaunchCoordinatorDelegate: AnyObject {
 }
 
 // Manages different types of onboarding that gets shown at the launch of the application
-class LaunchCoordinator: BaseCoordinator, OpenURLDelegate {
```

Comment:
The `LaunchCoordinatorDelegate` should not have `didRequestToOpenInNewTab` as well then (this means removing `didRequestToOpenInNewTab`conformance in BrowserCoordinator, SceneCoordinator and their respective tests/mock as well 🙏

Reply:
I think we'll also be able to remove this [`didFinishLaunchCompletion`](https://github.com/mozilla-mobile/firefox-ios/blob/cfb0689be5b8faf7efd5d9ffc80ba74c84682630/Client/Coordinators/Scene/SceneCoordinator.swift#L15-L16), since this was added to be able to test the `didRequestToOpenInNewTab` from `LaunchCoordinatorDelegate` in `SceneCoordinator`